### ISSUE TITLE
Please Remove the University of Kansas DNS Servers

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jeevandongre/namebench#44]
```
We would not like any of the University of Kansas DNS servers listed as
"public servers" in the namebench app even though many do allow recursive
queries to the world.  All DNS servers in the 129.237.0.0/16 address space
are intended to be used by University staff, faculty, students, and
affiliates only.

Please remove the following:

129.237.32.1 (finch.cc.ku.edu)
129.237.133.1 (kuhub.cc.ku.edu)

As well as anything else in the 129.237.0.0/16 address space.


Thanks,

--------------------------------------------
Brett Becker, CISSP
Enterprise IT Architect, Enterprise Architecture & Integration 
Information Technology - A Division of Information Services
The University of Kansas
--------------------------------------------



```

Original issue reported on code.google.com by `brettbec...@gmail.com` on 8 Dec 2009 at 2:43

**Comment by GoogleCodeExporter** [comment]
```
They've been removed as part of revision 425. This change will be part of the 
1.0.2 
release which will be released within the next 2 days. Thanks!
```

Original comment by `thomas.r...@gmail.com` on 8 Dec 2009 at 3:40
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.0.2 (just released)

```

Original comment by `tstromb...@google.com` on 8 Dec 2009 at 7:33
- Changed state: **Fixed**
